Bottom: 3d655f5243c68e2800f0f151a612cdeb84f1f0d7
Top:    7ea14d73ceb1423f178d20b2dc4328aab0c65c05
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-03 21:53:02 +0000

Refresh of bentley-ottann-test-implementa

---

diff --git a/src/Makefile.am b/src/Makefile.am
index 55d14c7..8f57701 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -37,6 +37,7 @@ PCB_SRCS = \
 	box.h \
 	buffer.c \
 	buffer.h \
+	cairo.cairo.h \
 	cairo/cairoint.h \
 	cairo/cairo-atomic-private.h \
 	cairo/cairo-bentley-ottmann.c \
